### ISSUE TITLE
Let's sort out global install recommendations.

### ISSUE THREAD

**Issue by bobthecow** [bobthecow/psysh#170]
`composer global require` is sub-optimal, because it puts psysh and all its dev and non-dev dependencies in a pile with anything else that's globally installed. It turns out that this approach doesn't scale. So maybe we shouldn't do that.

What should we do instead?

See #169

**Comment by GrahamCampbell** [comment]
Maybe we should recommend using a phar file, and adding it to your PATH in windows, or the equivalent for other OSs.

**Comment by Markcial** [comment]
Hi, maybe an import function encapsed on the Psysh namespace might help, like the ones encapsed on the `functions.php` file.

in pseudocode might look like this :
```php
namespace Psy;

function import($library) // library vendor Ex: "psy/psysh"
{
   static $loader;
   if (isset($loader)) {
     $loader = new Loader();
   }
   // when looking for library, check for its composer file with the autoload data, if library found but without composer warn about this.
   // look for local dependencies in vendor folder, return on success
   // look for global dependencies in the home folder, return on success
   // look for phar dependencies finally, return on success.
   // throw warning if libraries not found.
}
```
for the loader you might use one encapsed in the psysh namespace with the full functionalities, you can check out [this one](https://github.com/Markcial/stego/blob/master/src/Stego/Loader.php), feel fre to use it changing the name. It's ok for me.

This way, you can do a file like this one :
```php

require 'psysh.library';

use Psy;
import('symfony/console');

$console = new \Symfony\Component\Console\Application();
```
or
```php
require 'psysh.library';

Psy\import('symfony/console');

$console = new \Symfony\Component\Console\Application();
```

**Comment by bobthecow** [comment]
This was fixed with the "bin launcher" support in v0.5.0. Now, you can install globally and locally, and it will run the version you have installed locally, even if you just do `psysh` to call the global phar or composer installation.
